### PR TITLE
Master

### PR DESCRIPTION
**Pull request by jcriadop**
pagination fixed
update from old version fixed
track number fixed
translated vars to english

### CODE REVIEW

File: upgrade/upgrade-1.3.php [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+function upgrade_module_1_3($module)
+{
+	$configs = DB::getInstance()->executeS('SELECT * FROM '._DB_PREFIX_.'configuration WHERE `name` LIKE "%ZELERIS%"');
+	if (is_array($configs))
+		foreach($configs as $config)
+			$config_installed[] = $config['name'];
+	
+	if (in_array('ZELERIS_DESCRIPCION_MERCANCIA',$config_installed))
+	{
+		$sql='UPDATE '._DB_PREFIX_.'configuration SET `name` = "ZELERIS_MERCHANDISE_DESCRIPTION" WHERE `name`="ZELERIS_DESCRIPCION_MERCANCIA"';
```
Comment (Quetzacoalt91) [inline upgrade/upgrade-1.3.php]: You should update your data like this (assuming you want to remove the old fields:

```php
Configuration::updateValue('ZELERIS_MERCHANDISE_DESCRIPTION', Configuration::get('ZELERIS_DESCRIPCION_MERCANCIA'));
Configuration::deleteByName('ZELERIS_DESCRIPCION_MERCANCIA');
```

File: zeleriscarrier.php [hunk 2]
```diff
@@ -41,7 +41,7 @@ public function __construct()
 			$this->id_carrier = '';
 			$this->name = 'zeleriscarrier';
 			$this->tab = 'shipping_logistics';
-			$this->version = '1.2.5';
+			$this->version = '1.3';
```
Comment (Quetzacoalt91) [inline zeleriscarrier.php]: The new version should be 1.3.0

File: config.xml [hunk 3]
```diff
@@ -1,12 +1,11 @@
...
-	<tab><![CDATA[shipping_logistics]]></tab>
-	<is_configurable>1</is_configurable>
-	<need_instance>1</need_instance>
-	<limited_countries></limited_countries>
-</module>
\ No newline at end of file
+        <module>
+            <name>zeleriscarrier</name>
+            <displayName><![CDATA[Zeleris transporte]]></displayName>
+            <version><![CDATA[1.3]]></version>
```
Comment (Quetzacoalt91) [inline config.xml]: The new version should be 1.3.0

File: upgrade/upgrade-1.3.php [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+* @author    Línea Gráfica E.C.E. S.L.- https://www.lineagrafica.es/ <soporte@lineagrafica.es>
+* @copyright 2007-2015 TELEFONICA SERVICIOS INTEGRALES DE DISTRIBUCIÓN S.A.U.
+* @license   http://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
+* International Registered Trademark & Property of TELEFONICA SERVICIOS INTEGRALES DE DISTRIBUCIÓN S.A.U.
+*/
+
+if (!defined('_PS_VERSION_'))
+	exit;
+
+function upgrade_module_1_3($module)
```
Comment (Quetzacoalt91) [inline upgrade/upgrade-1.3.php]: The function name should be ```upgrade_module_1_3_0```

File: upgrade/upgrade-1.3.php [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+		}
+		
+		if (!in_array('packages',$field_installed))
+		{
+			$sql='ALTER TABLE '._DB_PREFIX_.'zeleris_orders ADD packages int(11) NOT NULL';
+			Db::getInstance()->execute($sql);
+		}
+	}
+	
+	return $module;
```
Comment (Quetzacoalt91) [inline upgrade/upgrade-1.3.php]: You must return a boolean. `true` if the upgrade succeed, or `false` if something went wrong.

**Comment by Quetzacoalt91** [comment]
Please also rename your file *upgrade-1.3.php* => *upgrade-1.3.0.php*

**Comment by jcriadop** [comment]
File renamed

**Comment by Quetzacoalt91** [comment]
Hi @jcriadop,

I do not understand what you did in you last commit. Do you know that the upgrade file will be executed one time ? So, if you return directly after the first SQL request execution, you will never be abble to apply the other checks.

Best regards

**Comment by jcriadop** [comment]
Hi, sorry for mistake, please review now.

Best regards.

**Comment by Quetzacoalt91** [comment]
Hi @jcriadop,

Thanks, your PR has been _manually_ merged.
Even if it is not shown as merged, your commits appear in the master branch: https://github.com/PrestaShop/zeleriscarrier/compare/914df27a670667eee548702dfc748605f4351a6f...f0b991aa9d0ee24aaca718f75c86d7627453ab42

I rebased them in order to remove `Revert "Revert "Revert "Revert "..."""""`.

Best regards

**Comment by jcriadop** [comment]
Ok, thanks.

**Pull request closed without merging**